kimchi/src/circuits/expr.rs

```diff
@@ -2263,7 +2267,7 @@ pub mod test {
                 gamma: one,
                 joint_combiner: None,
                 endo_coefficient: one,
-                mds: vec![vec![]],
+                mds: &Vesta::sponge_params().mds,
```

Comment:
I guess we'll be able to get rid of that field also, but we can figure that out in another PR :o

Reply:
I think it will just require to make `Enviroment` generic over the curve instead of the field, I will look into it.

Reply:
probably not a priority though, maybe we can make it an issue for now :o

Reply:
It can be done, but I'm not sure if it's worth adding the generics just for that field, probably best to let it for later.